gio/src/auto/app_launch_context.rs

```diff
@@ -59,11 +59,15 @@ pub trait AppLaunchContextExt: IsA<AppLaunchContext> + 'static {
 
     #[doc(alias = "g_app_launch_context_get_startup_notify_id")]
     #[doc(alias = "get_startup_notify_id")]
-    fn startup_notify_id(&self, info: &impl IsA<AppInfo>, files: &[File]) -> Option<glib::GString> {
+    fn startup_notify_id(
+        &self,
+        info: Option<&impl IsA<AppInfo>>,
```

Comment:
Breaking change but as we discussed we'd backport these now because of correctness

Reply:
Yep, it would be nice to backport it actually as it blocks some work we have in ashpd :)

Reply:
Will do